src/utils/haulProgressBar.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ *
+ * haulProgressBar.js
+ *
+ * @flow
+ */
+
+const multiProgress = require('multi-progress');
+const chalk = require('chalk');
+
+const progressBarFactory = new multiProgress(process.stderr);
+
+const progressBar = {};
+const lastPercent = {};
+const barOptions = {
+  complete: chalk.bold('='),
+  incomplete: ' ',
+  width: 20,
+  total: 100,
+};
+
+function createBarFormat(plat: string) {
+  const platform =
+    plat === 'android'
```

Comment:
This is potential issue in the future when we add more platforms. Is there any chance we can just capitalise `plat` passed?

Reply:
(let's see how it's being done in different places)